smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftDependency.kt

```diff
@@ -33,7 +33,7 @@ enum class SwiftDependency(
         "main",
         "0.1.0",
         "https://github.com/awslabs/smithy-swift",
-        Resources.computeAbsolutePath("smithy-swift/Packages", "Packages", "SMITHY_SWIFT_CI_DIR") + "/Packages",
+        Resources.computeAbsolutePath("smithy-swift", "", "SMITHY_SWIFT_CI_DIR"),
```

Comment:
Sorry @ganeshnj , I made these changes in #443 but I didn't push to get them reviewed & merged.

(Same as PackageManifestGenerator.kt above)